msteams-platform/toolkit/debug-background-process.md

```diff
@@ -101,11 +101,14 @@ When you select **Start Debugging (F5)**, the Teams Toolkit output channel displ
 
 In the set-up process, Teams Toolkit prepares the following registrations and configurations for your Teams app:
 
-1. [Registers and configures Microsoft Azure Active Directory(Azure AD) app](#registers-and-configures-microsoft-azure-active-directoryazure-ad-app)
-
-1. [Registers and configures bot](#registers-and-configures-bot).
-
-1. [Registers and configures Teams app](#registers-and-configures-teams-app).
+- [Debug background process](#debug-background-process)
+  - [Verification of prerequisites](#verification-of-prerequisites)
+  - [Register and configure Teams app](#register-and-configure-teams-app)
+    - [Registers and configures Microsoft Azure Active Directory(Azure AD) app](#registers-and-configures-microsoft-azure-active-directoryazure-ad-app)
+    - [Registers and configures bot](#registers-and-configures-bot)
+    - [Registers and configures Teams app](#registers-and-configures-teams-app)
+  - [Take a tour of your app source code](#take-a-tour-of-your-app-source-code)
+  - [See also](#see-also)
```

Comment:
Are these additional items (and the removal of order list markdown) intentional?

Reply:
these were not my changes - think they might have been auto-gen?

Reply:
I'm not sure where that text is from, but [latest source of this doc](https://raw.githubusercontent.com/MicrosoftDocs/msteams-docs/main/msteams-platform/toolkit/debug-background-process.md#L104-105) is the text in pink that's being changed here.